decidim-core/app/uploaders/decidim/avatar_uploader.rb

```diff
@@ -8,11 +8,11 @@ class AvatarUploader < ImageUploader
     process :validate_dimensions
 
     version :big do
-      process resize_and_pad: [500, 500]
+      process resize_to_fit: [40, 40]
```

Comment:
I needed to change this to `resize_to_fit`, otherwise the avatar was distorted. This version was previously unused, now I'm using it for the profile page, and optimized the size.

Reply:
I don't really know why we didn't spot this bafore, but we should be using double resolution sizes (probably 80x80 in this case) to comply with retina displays.